lib/valkyrie/persistence/active_fedora/orm/resource.rb

```diff
@@ -52,6 +53,14 @@ class Resource < ActiveFedora::Base
     property :nested_resource, predicate: ::RDF::URI("http://example.com/nested_resource"), class_name: "Valkyrie::Persistence::ActiveFedora::ORM::NestedResource"
     accepts_nested_attributes_for :nested_resource
 
+    def size=(size)
+      file_size = size
+    end
+
+    def size
+      file_size
+    end
+
```

Comment:
This is a workaround for `size` being a reserved word — it won't let you define a `size` property, but methods are fine.